src/clients/RTorrent.ts

```diff
@@ -312,6 +318,7 @@ export default class RTorrent implements TorrentClient {
 					`d.directory_base.set="${directoryBase}"`,
 					`d.custom1.set="${TORRENT_TAG}"`,
 					`d.custom.set=addtime,${Math.round(Date.now() / 1000)}`,
+					`d.custom2.set=paused,${paused}`,
```

Comment:
for rTorrent, we control paused vs not by using `load` vs `load.start` on line 315.